modules/op-starting-pipelines.adoc

```diff
@@ -38,17 +38,17 @@ Select *Add Credentials* if you want to specify an additional *Registry Server A
 +
 You can add multiple secrets based upon the number of resources in your Pipeline.
 
-. Click *Start* to start the PipelineRun.
-. The *Pipeline Run Details* page displays the Pipeline being executed. After the Pipeline starts, the Tasks and Steps within each Task are executed.
-You can:
-
-* Hover over the Tasks to see the time taken for the execution of each Step.
-* Click on a Task to see logs for each of the Steps in the Task.
-* Click the *Logs* tab to see the logs according to the execution sequence of the Tasks and use the *Download* button to download the logs to a text file.
+. Click *Start* to start the pipeline run.
+. The *Pipeline Run Details* page displays the pipeline being executed. After the pipeline starts, the tasks and steps within each task are executed. You can:
```

Comment:
Just a small suggestion. You may rephrase it as:
The *Pipeline Run Details* page displays that the pipeline is executed. 

If this doesn't align with the pitch or meaning you trying to intend, then leave this suggestion.

Reply:
@rishumehra  I suggest, not to change it because, when you start a pipeline, you will be redirected to the pipelineRun Details page which will be executing.. It takes its own time to finish the execution.